tfx/dsl/component/experimental/kubeflow_container_component.py

```diff
@@ -0,0 +1,106 @@
+# Lint as: python2, python3
```

Comment:
Drive by comments:

We shall also start to drop `# Lint as: python2, python3` from now for Python3 has become the default linter config recently.